src/Unleash/Utilities/FetchingToggleBootstrapUrlFailedException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Unleash.Utilities
+{
+    public class FetchingToggleBootstrapUrlFailedException : Exception
```

Comment:
Is it possible/sensible to make this inherit from UnleashException? It seems like everything else in the code uses that and it'd be nice for consumers to only have to worry about a single exception type (if that's rational, you know your code best)

Reply:
Yup defo, good catch!